src/site/stages/build/process-cms-exports/transformers/outreach-asset.js

```diff
@@ -0,0 +1,30 @@
+const { get } = require('lodash');
+
+const transform = entity => {
+  const modifiedEntity = {};
+
+  // Derive the raw entity properties.
+  const changed = get(entity, 'changed[0].value');
+  const status = get(entity, 'status[0].value');
+  const title = get(entity, 'title[0].value');
+  const fieldBenefits = get(entity, 'field_benefits[0].value');
+  const fieldDescription = get(entity, 'field_description[0].value');
+  const fieldFormat = get(entity, 'field_format[0].value');
+
+  // Add changed.
+  modifiedEntity.changed = new Date(changed).getTime() / 1000;
+
+  // Add entityBundle.
+  modifiedEntity.entityBundle = 'outreach-asset';
+
+  // Add remaining entity properties.
+  modifiedEntity.status = status;
+  modifiedEntity.title = title;
+  modifiedEntity.fieldBenefits = fieldBenefits;
+  modifiedEntity.fieldDescription = fieldDescription;
+  modifiedEntity.fieldFormat = fieldFormat;
```

Comment:
This currently is missing `fieldMedia` as I wanted to get thoughts on how we should add references to other entities within our transform functions 🤔

Reply:
I guess I don't know what reference you're talking about. Transformation happens after entities are dereferenced (expanded), so you shouldn't have to do anything fancy. Actually, the nested `fieldMedia` will already be dereferenced _and_ transformed by the time this gets here.

Reply:
All "child" entities should be filtered & transformed by the time that we are trying to transform their parent entity.

Reply:
Ahh got it 👍